lib/LaTeXML/Common/Model/RelaxNG.pm

```diff
@@ -551,6 +551,7 @@ sub cleanTeX {
   $string =~ s/\#/\\#/g;
   $string =~ s/<([^>]*)>/\\texttt{$1}/g;    # An apparent convention <sometext> == ttfont?
   $string =~ s/_/\\_/g;
+  $string =~ s/\*/any/g;                    # ????
```

Comment:
unsure about this comment, what is a sample input that has the star?

Reply:
it's for `rawhtml` which can take *any* xhtml.